### PR TITLE
Tests for #5 have been completed

### LINKED ISSUE #5: Create test file for 'extract' python file
Description:
Write tests so that we can follow the Test Driven Development (TDD) principle for best practice.

Outcome:
File with tests that gives a pytest --cov of minimum 60%.

User Story:
As developers, we need to create tests so that we know our code is bug-free and resilient around any edge cases and thus ensures a good user experience.

Design:
Minimum 5 tests?

### PR DESCRIPTION
**Pull request by keoghrmj**
Created simple tests for extract.py

Close #5

### CODE REVIEW

File: pipeline/.coverage
Comment (SurinaCS) [inline pipeline/.coverage:1]: maybe add to gitignore

File: pipeline/test_extract.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+import pytest
+from unittest.mock import patch, mock_open
+
+from extract import (extract_botanist_information,
+                     extract_location_information,
+                     extract_metric_information,
+                     extract_plant_information,
+                     fetch_and_collect_data,
+                     write_to_csv)
```
Comment (SurinaCS) [inline pipeline/test_extract.py:12]: nice formatting

File: pipeline/test_extract.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+def test_extract_plant_information():
+    input_data = {"name": "Test Test", "scientific_name": ["Test Test"],
+                  "images": {"original_url": "https://test.jpg"}}
+    output_data = {"name": "Test Test", "scientific_name": "Test Test",
+                   "original_url": "https://test.jpg"}
+    assert extract_plant_information(input_data) == output_data
+
+
+@patch("extract.requests.get")
```
Comment (SurinaCS) [inline pipeline/test_extract.py:79]: nice use of patching!

**Review by qrafiq1 — APPROVED** [review]
Looks good

**Review by SurinaCS — APPROVED** [review]
nice
